templates/repo/clone_script.tmpl

```diff
@@ -24,6 +25,10 @@
 		const btn = isSSH ? sshBtn : httpsBtn;
 		if (!btn) return;
 
+		if (methodDesc) {
+			methodDesc.innerText = btn.dataset.description;
```

Comment:
https://docs.gitea.com/contributing/guidelines-frontend#html-attributes-and-dataset